src/common/hoc/IsLoadingHOC.tsx

```diff
@@ -1,55 +1,47 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { useEffect, useState } from "react";
-import { Spinner } from "reactstrap";
+import React, { useState } from "react";
+import { Alert, Spinner } from "reactstrap";
 
+// im havin an issue here
+// export const IsLoadingHOC = (WrappedComponent: React.FunctionComponent, loadingMessage: string) => {
 export const IsLoadingHOC = (WrappedComponent: any, loadingMessage: string) => {
```

Comment:
If possible I think it would be best to not use this custom HOC. I would look into React Suspense.